css/index.css

```diff
@@ -31,6 +31,33 @@ footer, header, hgroup, menu, nav, section {
 body {
 	line-height: 1;
 }
+
+.Header {
+    height: 250px;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.Header .HeaderNav {
```

Comment:
What would you need to change so that the bottom of the lambda img & the bottom of the nav's anchor tags are aligned?